app/src/main/java/ch/epfl/sdp/appart/AdCreationActivity.java

```diff
@@ -86,13 +88,15 @@ private void createAd() {
                     Snackbar.LENGTH_SHORT).show();
             return;
         }
-        // TODO add back when new loading image system is finished
-        /*if (!mViewModel.hasPhotos()){
-            Snackbar.make(findViewById(R.id.horizontal_AdCreation_scrollView),
-                    getResources().getText(R.string.snackbarNoPhotos_AdCreation),
-                    Snackbar.LENGTH_SHORT).show();
-            return;
-        }*/
+
+        if (picturesUris == null || picturesUris.size() == 0) {
+            picturesUris = new ArrayList<>();
+            Resources resources = getApplicationContext().getResources();
+            picturesUris.add(Uri.parse(ContentResolver.SCHEME_ANDROID_RESOURCE + "://" +
+                    resources.getResourcePackageName(R.drawable.blank_ad) + "/" +
+                    resources.getResourceTypeName(R.drawable.blank_ad) + "/" +
+                    resources.getResourceEntryName(R.drawable.blank_ad)));
+        }
```

Comment:
You could use `FirebaseLayout.SEPARATOR` for this

Reply:
I second this, and you will have to run cirrus again in any case since the branch is out of date ahah

Reply:
:((